### PR TITLE
Create 2020-11-05-this-month-in-solid.md

### PR DESCRIPTION
**Pull request by MarrelleBailey**
Newsletter for November Solid World

### CODE REVIEW

File: _posts/newsletter/2020-11-05-this-month-in-solid.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+05 November 2020 – This month in Solid, we had a lot of movement in the community and got to hear about some of those activities in detail during November’s Solid World. Our theme for the month was to learn how Solid is delivered at a national scale, focusing on Belgium. But first, we heard a few updates from around the ecosystem.
+
+Recently, there was a change in the Solid Community Pod server. During Solid World, Tim Berners-Lee confirmed that the service is back up and running at solidcommunity.net. There are a few members in the Solid community to thank for helping with this: Justin Bingham, Jackson Morgan, Jeff Zucker, Alain Bourgeois, Eric Prud'hommeaux, and Michiel de Jong. 
+
+As usual, Ruben Verborgh shared a progress update on the Community Solid Server, focusing on impressive new database flexibility. The recording for the full demo is coming soon. And Osmar (Oz) Olivo gave a preview of big news about Inrupt’s roadmap for bringing Solid to developers and organizations. You can keep up with the latest on Inrupt’s blog.
```
Comment (Vinnl) [inline _posts/newsletter/2020-11-05-this-month-in-solid.md:18]:
```suggestion
As usual, Ruben Verborgh shared a progress update on the Community Solid Server, focusing on impressive new database flexibility. The recording for the full demo is coming soon. And Osmar (Oz) Olivo gave a preview of big news about Inrupt’s roadmap for bringing Solid to developers and organizations. You can keep up with the latest on [Inrupt's blog](https://inrupt.com/blog).
```

File: _posts/newsletter/2020-11-05-this-month-in-solid.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Our first presenter for November’s Solid World was Frédéric Hennequin, who is the Solution Architecture for My Citizen Profile and the Webplatform program of Information Agency of Flanders. We got to see a demo video of how your pod (or profile) would work, from logging into your account, gathering banking information, and using autofill from various accounts on the vlandeer.be website. Learning of the versatility of how much can be done with Solid was very exciting to see, and Flanders has plans to add even more. 
+
+Our final presenter was Pieter Heyvaert, the Development Lead and Developer Advocate at Ghent University - IDLab, imec. He discussed the creation of a job application service for those with dyslexia or dyscalculia. The frontend is easy to use and understand by giving the user various ways to upload or input their information and resume by audio, video, or text in addition to a reader option. Lastly, he explained why they are using Solid on the backend, and how data flows are architected. 
+
+Register for Solid World December here, and keep an eye out for agenda updates over the coming weeks.
```
Comment (Vinnl) [inline _posts/newsletter/2020-11-05-this-month-in-solid.md:26]:
```suggestion
[Register for Solid World December here](https://www.eventbrite.com/e/solid-world-tickets-128665029567), and keep an eye out for agenda updates over the coming weeks.
```

File: _posts/newsletter/2020-11-05-this-month-in-solid.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+### [Press](https://solidproject.org/press)
+
+* 
+
+### In Other News
+
+#### Solid Implementations
+
+*
+
```
Comment (Vinnl) [inline _posts/newsletter/2020-11-05-this-month-in-solid.md:38]: If there's nothing here we might as well delete this section, I think?

```suggestion
```

**Review by Vinnl — APPROVED** [review]
Suggested a couple of links and to remove empty headings (press "Commit suggestion" under the suggestions to apply them if you agree), otherwise looks good to me.

**Comment by NSeydoux** [comment]
This all looks good, but @Vinnl I have a doubt: should we merge this in `staging` first, or can it go directly to `master` ? I think it was agreed that some changes could be pushed directly to `master`, but I'm not 100% sure ^^

**Comment by Vinnl** [comment]
@NSeydoux `staging`'s gone - you can follow the `netlify/solidproject-dot-org/deploy-preview` "details" link below to see what it will look like. It's just that who is allowed to approve/merge it differs per the scope of the change. Since TMiS is a relatively standard format that usually already went out to all email list subscribers, I considered that to be a minor change so far that does not need editorial approval. So far haven't heard any complaints about that, so I suppose we can merge :)

**Pull request merged**